test/homepage-noweb-test.js

```diff
@@ -68,6 +68,30 @@ suite.addBatch({
                 assert.ifError(err);
                 br.assert.success();
                 br.assert.text("H1", "pump.io API");
+            },
+            "it not has login or register buttons": function(err, br) {
+                br.assert.success();
+                br.assert.evaluate('document.getElementById("login")', null);
+                br.assert.evaluate('document.getElementById("register")', null);
+            }
+        },
+        "and we request javascript file": {
+            topic: function() {
+                var cb = this.callback,
+                    browser = new Browser();
```

Comment:
Don't use Zombie for this, just use a plain HTTP HEAD request (see `test/lib/http.js` for some useful stuff, IIRC)

Reply:
I will try.

Reply:
why not Zombie?

Reply:
It's just not needed and introduces more complexity. A plain HTTP request will be faster and easier to debug.

Also IIRC Zombie has issues where it'll treat _any_ 4xx response as an error, even if that's what you're trying to assert on - but I think we may have upgraded past the version that did that so if you didn't see that locally, that'd be why :D

Reply:
If you want an example `test/xss-ui-test.js` is pretty easy to understand :thumbsup:

Reply:
Ok, thanks